db/seeds.rb

```diff
@@ -6,23 +6,34 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-categories = Category.create([{title: "Apparel"},{title: "Snowboards"}])
+Category.create([{ title: "Apparel" },{ title: "Snowboards" }])
 
-apparel = Category.first.items.create([{name: "Hoody",
-                      description: "Keeps you slightly warm",
-                      price: 80},
+Category.first
+        .items
+        .create([{ name: "Hoody",
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 8) spaces for indenting an expression spanning multiple lines.